jdaviz/app.py

```diff
@@ -2048,15 +2047,28 @@ def _create_data_item(data):
             typ = 'cube'
         else:
             typ = 'unknown'
+
         # we'll expose any information we need here.  For "meta", not all entries are guaranteed
         # to be serializable, so we'll just send those that we need.
+
+        def _expose_meta(key):
+            """
+            whether to expose the metadata to the data-item in the UI based on the dictionary key
```

Comment:
Is this accurate?

```suggestion
            Whether to expose the app metadata to the data-item in the front-end layer based on the dictionary key.
            This does not affect the Metadata Viewer plugin.
```

Reply:
I wouldn't consider it app-metadata.  How about "Whether to expose this metadata entry from the glue data object to the vue-frontend via the data-item, based on the dictionary key."

Reply:
Is "so that we can control their values in the front-end" accurate or is it for something else? A new dev needs to be able to understand enough to use this in the future.

Reply:
In this case, that is why, yes, but its really anything from the data's metdata that we need to access from vue for _any_ reason.

Reply:
ok, I'll add in-line comments for the individual exceptions that are currently exposed as examples.